<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="engine_files_tree_view">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="engine_file_name_browser">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>540</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit_button">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>540</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="engine_lable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>| Engine Files</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QTreeView" name="theme_files_tree_view">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QLabel" name="theme_lable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>| Theme Files</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="theme_file_name_browser">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QLabel" name="unit_lable">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>| Units</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="selected_engine_lable">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>| Selected Engine File</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="selected_theme_lable">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>| Selected Theme File</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="temp">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>**********************
* To Be Developed... *
**********************</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
